src/test/resources/org/prebid/server/ApplicationTest/openrtb2/test-auction-response.json

```diff
@@ -410,116 +414,116 @@
       "httpcalls": {
         "adform": [
           {
-            "uri": "{{ adform.exchange_uri }}/?CC=1&rp=4&fd=1&stid=tid&ip=192.168.244.1&adid=ifaId&bWlkPTE1",
+            "uri": "http://localhost:8090/adform-exchange/?CC=1&rp=4&fd=1&stid=tid&ip=192.168.244.1&adid=ifaId&bWlkPTE1",
```

Comment:
This change was done by mistake I presume

Reply:
Yes, thanks i'll check all others.